crypto/ec/ecx_meth.c

```diff
@@ -163,25 +279,33 @@ static int ecx_priv_decode(EVP_PKEY *pkey, const PKCS8_PRIV_KEY_INFO *p8)
         plen = ASN1_STRING_length(oct);
     }
 
-    rv = ecx_key_op(pkey, pkey->ameth->pkey_id, palg, p, plen, X25519_PRIVATE);
+    rv = ecx_key_op(NULL, pkey, pkey->ameth->pkey_id, palg, p, plen,
+                    KEY_OP_PRIVATE);
     ASN1_OCTET_STRING_free(oct);
     return rv;
 }
 
 static int ecx_priv_encode(PKCS8_PRIV_KEY_INFO *p8, const EVP_PKEY *pkey)
 {
-    const X25519_KEY *xkey = pkey->pkey.ptr;
+    unsigned char *privkey;
     ASN1_OCTET_STRING oct;
     unsigned char *penc = NULL;
     int penclen;
 
-    if (xkey == NULL || xkey->privkey == NULL) {
+    if (pkey->pkey.ptr == NULL) {
+        ECerr(EC_F_ECX_PRIV_ENCODE, EC_R_INVALID_PRIVATE_KEY);
+        return 0;
+    }
+
+    GETPRIVKEY(privkey, pkey);
+
```

Comment:
Remove this blank line, same reason as above.

Reply:
As above, this got moved around a bit anyway as part of the pkey.ptr change.